public/app/features/explore/Logs.tsx

```diff
@@ -260,13 +260,19 @@ class UnthemedLogs extends PureComponent<Props, State> {
     if (this.props.onStartScanning) {
       this.props.onStartScanning();
     }
+    reportInteraction('grafana_explore_logs_scanning_start_clicked', {
```

Comment:
I am wondering if we want to do `grafana_explore_logs_scanning_button_clicked` and do type start/stop?  WDYT?

Reply:
good idea, changed 👍